corehq/apps/app_manager/static/app_manager/js/modules/report_module.js

```diff
@@ -395,8 +405,8 @@ hqDefine('app_manager/js/modules/report_module', function () {
     });
 
     return {
-        ReportModule: ReportModule,
-        StaticFilterData: StaticFilterData,
+        reportModuleModel: reportModuleModel,
+        staticFilterDataModel: staticFilterDataModel,
```

Comment:
What do each of these case syntaxes represent?  The object inheritance pattern?  I'd expect variables with initial lowercase characters to be an object instance, not a constructor, but that's just because that's how Python does it.

Reply:
The lowercase first is for functional inheritance, ucfirst for classical inheritance. Check out discussion here: https://github.com/dimagi/commcare-hq/pull/19895#pullrequestreview-106915955